Adds documentation and sample app for msal go (#22)
Azure-Samples/msal-managed-identity: src/go/samples/AppService/overview.md

---
services: active-directory
platforms: go
author: andrewohart
level: 400
client: Azure functions
service: Azure Key Vault
endpoint: Microsoft identity platform
page_type: sample
languages:
  - go  
products:
  - azure
  - azure-active-directory  
  - go
  - managed identity
  - key vault
description: "This sample showcases how to use the App Service Source and develop an Azure function that gets a secret from a key Vault using Managed Identities."
---
# App Service

This sample showcases the [App Service source](https://learn.microsoft.com/en-us/azure/app-service/) and how to acquire a secret from an [Azure Key Vault](https://vault.azure.net) using Azure Managed Identity  
We will cover [Azure Functions](https://learn.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-other?tabs=go%2Cwindows) and how they can be used to return a token from the key vault

## Topology

![Overview](../../images/call-kv.png)

### Scenario

> You want to access an Azure Key Vault secret from a function deployed in Azure. And, you don't want to worry about managing secrets or app credentials.

Reviewer: ```suggestion
Access an Azure Key Vault secret from a function deployed in Azure without managing secrets or app credentials.
```
